documentation/vespa-quick-start-centos.html

```diff
@@ -128,22 +128,24 @@ <h2 id="next-steps">Next Steps</h2>
 
 <ul>
   <li>
+    This application is fully functional and production ready, but you may want to
+    <a href="multinode-systems.html">add more nodes</a> for redundancy.
+  <li>
     Try the <a href="tutorials/blog-search.html">Blog search and recommendation tutorial</a>
     to learn more about using Vespa
-  </li><li>
+  <li>
     See <a href="jdisc/developing-applications.html">developing applications</a>
     on adding your own Java components to your Vespa application.
-  </li><li>
+  <li>
     <a href="api.html">Vespa APIs</a> is useful to understand how to interface with Vespa
-  </li><li>
+  <li>
     Explore the <a href="https://github.com/vespa-engine/sample-apps/tree/master">
       sample applications</a>
-  </li><li>
+  <li>
     How to <a href="securing-your-vespa-installation.html">secure a Vespa installation</a>
-  </li><li>
-    Try the <a href="vespa-quick-start-multinode-aws.html">AWS EC2 multinode quick start</a>
+  <li>
+    To run on AWS, see the <a href="vespa-quick-start-multinode-aws.html">AWS EC2 multinode quick start</a>
     and <a href="vespa-quick-start-multinode-aws-ecs.html">AWS ECS multinode quick start</a>
-  </li>
```

Comment:
i find my xcode editor some times confused in indenting html without closing tags, that is why I normally keep them